### ISSUE TITLE
Acebug makes right-side console prompt unusable

### ISSUE THREAD

**Issue by bobdobbs** [MikeRatcliffe/Acebug#14]
ff3.6.16
fb 1.6.2
ubuntu 10.10

Firebug console works fine if the prompt is sitting at the bottom, as is the default.
But if I click on the icon that moves the prompt to the right then the prompt becomes completely unresponsive.

If I disable acebug and then restart ff, a right-side vertically sitting prompt becomes responsive again.

**Comment by nightwing** [comment]
hi
to use acebug you need newer version of firebug (1.7X.0a10 or higher)
latest version is http://getfirebug.com/releases/firebug/1.7X/firebug-1.7X.0b3.xpi
(see also description on acebug's page on AMO)

**Comment by bobdobbs** [comment]
Thanks for the reply.

Alas, it seems that firebug1.7 isn't available for my version of ff yet.
(I'm running firefox3.6)

I'll have to wait for a compatiblity update before I proceed to play with Acebug.

**Comment by MikeRatcliffe** [comment]
It is available ... all that you need to do is click this link in Firefox:
http://getfirebug.com/releases/firebug/1.7X/firebug-1.7X.0b3.xpi

It should then install.

**Comment by bobdobbs** [comment]
When I do that, the installation process starts. Then I get a dialogue box with an error message telling me that this version of fb is not available for my version of firefox.

I'm running firefox 3.6.16pre.

This is the latest version available via the ubuntu repos.

**Comment by nightwing** [comment]
fbug1.7's minversion is really  4.012b now,
but it works with 3.6 too

in about config create preference 
&nbsp;&nbsp;&nbsp;&nbsp;extensions.checkCompatibility.3.6  &nbsp;&nbsp;&nbsp;&nbsp;  false

or use mrtechToolkit addon

this preference is usefull for other addons too since most addon incompatibilities are caused by min/maxVersion (lots of people can't use addons they want because of one silly number)

**Comment by bobdobbs** [comment]
Thanks for the pointers.

I created the preference.
The install failed with the same message. It looked like the setting was toggling back automatically to 'true'. Weird.

I looked a little closer and saw that there was an existing preference:
"extensions.checkCompatibility". This was set to true.
I set it to false.

I guess that if 'extensions.checkCompatibility' is set to 'true', then 
'extensions.checkCompatibility.3.6 becomes automatically set to true.
But I don't know.

When both are set to 'true', I was able to successfully install the xpi you pointed to.

After a restart, fb was working. But acebug was not.

I went to 'tools '-> 'addons'. Acebug was not enabled.
I hit the 'enabled' button. The dialogue came up, telling me that ff needed to restart.
I restarted.

Firebug was no longer available to me.
The icon is not visible. 

I go to 'addons' again. The 'enable' button for ff is greyed out. I cannot select it.
A red apostraphe icon is over the fb icon, and a message says that this version of ff is not  compatible with my version of ff.

But, hang on, this is interesting... 
If I right click on the ff section of the addons list, an item appears:
"make compatible"

I click on it and I see a dialogue titled: "Really change extension compatibility?"
"This will force the extension to try to work in a version of the application that it normally would not. This could cause the application to stop working correctly. Are you sure you want to do this?"

Aw man... I'm nervous enough already, with all the warnings I got about changing the config... but I'll try it anyway...

This time I don't get a message about having to restart ff, but the fb icon hasn't appeared on the bottom of the browser, so I restart manually...

After restart, there is no acebug. I check addons again, and see that it has been disabled again. So I enable it, and am prompted to restart.

After restart, I have both fb and acebug.

<exhale>

omg! Emacs keybindings too! I'm hooked.

Thanks for the help. For the moment it looks as if its all going smoothly.
If something explodes, I'll let you know.

Thanks.
